app/assets/javascripts/give_time.js.coffee

```diff
@@ -5,14 +5,14 @@ jQuery.validator.addMethod "either-hours-minutes-informed", ((value, element) ->
 giveTimeReadyFn = () ->
   config =
     submitHandler: (form) ->
-      amount = $("#transfer_hours").val() * 3600 + $("#transfer_minutes").val() * 60
-      $("#transfer_amount").val(amount)
+      containsInputFields = $(form).find("#transfer_hours, #transfer_minutes").length == 2
+      amount = $(" #transfer_hours ").val() * 3600 + $(" #transfer_minutes ").val() * 60
 
-      if amount > 0
-        form.submit()
-      else
-        $(form).find('.form-actions .error').removeClass('invisible').show()
```

Comment:
@Morantron why this check + front-end validations were removed? We introduced recently, in #421

Reply:
@Morantron @sseerrggii I think we should leave this front-end validation working for both scenatios: regular transfer and multi-transfers. We have back-end and DB constraints, but I'd like to keep this friendly i18n front-end validation :ok_hand:

Reply:
I've rearranged the code to preserve this behavior. I only need the bits that fill the hidden input with the total amount of seconds ( hours + minutes ). Validation logic breaks the multi transfer wizard, since we always render the field but it might not be filled yet.

Reply:
@Morantron I think it's important to keep this front-end validation, I just tried locally this branch and I get the following weird behavior: 

1) I fill with amount 0 (0 hours, 0 mins)
2) `Transfers created successfully` as a flash message
3) I see a db ROLLBACK in the log and the transfer isn't created at all

That's confusing, IMHO we should improve it before shipping it :) 

Maybe we can try something like: perform js validation only in specific step.